conf/ref_databases.config

```
@@ -306,6 +306,12 @@ params {
             citation = "McDonald, D., Price, M., Goodrich, J. et al. An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea. ISME J 6, 610–618 (2012). https://doi.org/10.1038/ismej.2011.139"
             fmtscript = "taxref_reformat_qiime_greengenes85.sh"
         }
+        'greengenes2022' {
```

I propose following the scheme in this config more closely, i.e.
```suggestion
        'greengenes2=2022.10' {
```
and add another entry with `greengenes2`? In case there are more versions expected